css/index.css

```diff
@@ -0,0 +1,102 @@
+body {
+    background-color: rgb(215, 234, 250);
+    color:black;
+    font-family: 'Times New Roman', Times, serif;
+}
+
+section {
+    padding: 3px;
+}
+
+h1 {
+    font-size: large;
+    font-weight: 600;
+    color:black;
+}
+
+h2 {
+    font-size: large;
+    font-weight: 600;
+    color:black;
+}
+
+/* styling experience list as block*/
+.list_experience {
+    display: block;
+    border-style: ridge;
+    padding: 5px;
+
+}
+
+
+
+/*styling blocks in a section*/
+.skills_class {
+    display: block;
+    border-style:ridge;
+    padding: 5px;
+   
+}
+
+/*styling form */
+.field_label {
+    display: block; 
+    font-size: medium;
+    color: darkslateblue;
+    font-weight: 600;
+    padding: 5px ;
+}
+
+/* button styling */ 
+
+.button { 
+    background-color:darkslategrey;
+    color: white; 
+    padding: 5px 5px;
+    text-align: center;
+    display: inline-block;
+    font-size: 5;
+}
+
+.button_remove { 
+    background-color:crimson;
+    color: white; 
+    padding: 5px 5px;
+    text-align: center;
+    display: inline-block;
+    font-size: 5;
+}
+
+.messages_section {
+    display: block;
+    background-color: azure;
+    color: black;
+    padding: 5px;
+    text-align: center;
+    width: 550px;
+    margin: left;
```

Comment:
If you want to add spacing on the left side of the element, you can use `margin-left` as the css property, followed by how much space you want: [https://developer.mozilla.org/en-US/docs/Web/CSS/margin-left](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-left)

If you don't want to write `margin-left`, `margin-right`, `margin-top`, and `margin-bottom` separately for one selector, you can combine them by using the shortcut `margin` and adding values going clockwise from the top:

`margin: 1px 2px 2px 3px;` will set 1px of space above the element, 2px of space to the right, 2px to the bottom, and 3px to the left.

Or if you do something like `margin: 1px` it will just add 1px of space to all four sides